src/interactive-window/editor-integration/codelensProviderActivator.ts

```diff
@@ -17,7 +17,7 @@ export class CodeLensProviderActivator implements IExtensionSyncActivationServic
 
     public activate() {
         this.extensionContext.subscriptions.push(
-            languages.registerCodeLensProvider([PYTHON_FILE_ANY_SCHEME], this.dataScienceCodeLensProvider)
+            languages.registerCodeLensProvider([PYTHON_FILE, PYTHON_UNTITLED], this.dataScienceCodeLensProvider)
```

Comment:
`export const PYTHON_FILE = { scheme: 'file', language: PYTHON_LANGUAGE };` we won't have codelens on `.dev` if we enforce `file` scheme.

Reply:
what scheme is used on .dev?

Reply:
reverted